### PR TITLE
Awesome books ES6

### PR DESCRIPTION
**Pull request by Sammy3000**
## In this pull request I did the following:
- Re-created my Awesome books app following the rules:
-Used ES6 syntax. Refactored my methods and functions to arrow functions.
-Divided my code into modules. Saved each module in a separate file in the modules dir.
-Display the current date below the navigation bar.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @Sammy3000,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a readme file. Kindly add a readme file using [this template](https://github.com/microverseinc/readme-template). Ensure all the headers are present.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      </div>
+      <table class="booksContainer">
+        <!-- <thead>
+          <tr>
+            <th>title</th>
+            <th>by</th>
+            <th>author</th>
+            <th></th>
+          </tr>
+        </thead> -->
```
Comment (FrederickMih) [inline index.html]: - [ ] Please let's avoid pushing commented codes to the repo. Comments render our codebase messy.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  font-weight: bold;
+  margin-left: 300px;
+  padding: 0.2rem;
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+::placeholder {
+  text-align: start;
+  color: black;
```
Comment (FrederickMih) [inline index.css]: - [Optional] Placeholder texts are too pronounced. What do you think of some gray color?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (FrederickMih) [inline README.md:153]: - [Optional] License Link is broken. Let's kindly fix it in order to render the README file professional :+1:

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @Sammy3000, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Descriptive README.md file :+1: 
- Add and remove book features still work after converting the app to use JS modules and ES6 :+1: 
- PR is professional :+1: 
- All Linter checks passed :+1: 


## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**